lib/utils.js

```diff
@@ -317,6 +317,17 @@ get_clean_first_word = function(user_message) {
         .toUpperCase();         // capitalise
 };
 
+// ID HELPERS
+
+/**: extract_id_dob(id)
+Extracts the date of birth from a South African identification number
+
+:param string id: id number
+*/
+extract_id_dob = function(id) {
```

Comment:
please make this `extract_za_id_dob` - this system is pan-Africa now, we may need extraction functions per country